test/tests/passwords.js

```diff
@@ -9,15 +9,26 @@ describe("Client passwords", function() {
 	it("hashed password should match", function() {
 		// Generated with third party tool to test implementation
 		let comparedPassword = Helper.password.compare(inputPassword, "$2a$11$zrPPcfZ091WNfs6QrRHtQeUitlgrJcecfZhxOFiQs0FWw7TN3Q1oS");
+		comparedPassword.then((result) => {
+			expect(result).to.be.true;
+		});
```

Comment:
Actually, this doesn't test anything. Whether you test against `true` or `false`, it passes.
With promises, you need to do `return comparedPassword.then(...);`.

Alternative will be to use `chai-as-promised` to be able to do `return expect(comparedPassword).to.eventually.be.true;` but we will add that later, not worth doing it for 3 tests only :)